static/src/javascripts/projects/common/modules/commercial/article-aside-adverts.js

```diff
@@ -32,7 +32,7 @@ define([
             $adSlotContainer;
 
         // are article aside ads disabled, or secondary column hidden?
-        if (!commercialFeatures.articleAsideAdverts || colIsHidden) {
+        if (!commercialFeatures.articleAsideAdverts || colIsHidden || config.hasTone('Match reports')) {
```

Comment:
Is this going to be here permanently? Perhaps there's a way we can handle it in commercialFeaturePolicies. Perhaps a tonePolicy that applies all of these little switches.